recipes/check_shapes/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "check_shapes" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b699fcb1e60bb17e2c97007e444b89eeeea2a9102ff11d61fb52454af5348403
+
+build:
+  number: 0
+
+outputs:
+  - name: check_shapes
+    build:
+      noarch: python
+    script: python -m pip install . -vv
```

Think we need this indented

```suggestion
    build:
      noarch: python
      script: python -m pip install . -vv
```